CONTRIBUTING.md

```diff
@@ -161,10 +161,17 @@ Procedure of changing the image for the gatsby-image is the same as we change fo
 ## Adding Images and Icons
 
   1. It is recommended to use `@react-icons/all-files` instead of `react-icons` for importing icons. The issue with react-icons is that even though we are importing a single icon from the package/folder it still imports the complete icon folder which is creating unwanted junks of JS in the build.
-  2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site. This performs automatic image optimization, thereby improving site performance.
+  2. It is recommended to use Gatsby's `<StaticImage>` instead of `<img>` tag to display static images on the site **except for SVG images**. This performs automatic image optimization, thereby improving site performance.
   3. Avoid creating duplicate copies of the same image under different folders. All images must be imported from `/assets/images` folder. 
 
 
+## Site Performance Considerations
+
+1. Analyze page load metrics of mobile version in lighthouse report (see status of GitHub workflow on your open PR).
+2. While creating a new page make sure to use Gatsby Head API for SEO.
+3. Prefer using `loading="eager"` for Hero images to improve LCP and FCP.
```

Comment:
Using eager will increase the page load time and that is not out preferred choice.
We are/should be using `lazy` as the option (default for Gatsby images). To avoid CLS we should always provide `width` and `height` attributes to the image.

Reply:
This is only for hero images. We want the FCP and LCP to load faster as this is the first image we will see when the page loads. Lazy is by default in Gatsby Image and best for all below the fold images

Reply:
Ahh....I missed the word hero, my bad.
Looks good.